pipeline/build/stages.groovy

```diff
@@ -138,14 +138,9 @@ def buildPackages() {
     }
   }
 
-  utils.checkoutRepo('versions', gitRepos)
-  dir('versions') {
-    // The commit hashes should be accessible from the GitSCM object
-    // https://issues.jenkins-ci.org/browse/JENKINS-34455
-    versions_repo_commit_id = sh(
-      script:"git rev-parse HEAD", returnStdout: true).trim()
-    echo versions_repo_commit_id
-  }
+  checkoutResult = utils.checkoutRepo('versions', gitRepos)
+  versionsRepoCommitId = checkoutResult['GIT_COMMIT']
+  echo(versionsRepoCommitId)
```

Comment:
Are we missing a `$` sign here?

Reply:
You mean in the `echo` call? No, it only needs to be used inside string, e.g.: `echo("$versionsRepoCommitId")`